File: java/Override/Override/src/Override/Override/src/DerivedClass.java
// 整数値, 文字列, 実数値, 真偽値を持つクラス
class DerivedClass extends BaseClass {	// BaseClassクラスを継承したDerivedClassクラスの定義.

	// フィールドの定義
	private float f;	// float型フィールドf
	private boolean b;	// boolean型フィールb

	// メソッドの定義
	// 初期化処理
	DerivedClass() {	// コンストラクタDerivedClass()
		super(0, "-");	// super(BaseClassのコンストラクタ)で, 0と"-"をセット.
		f = 0;			// fに0をセット.
		b = false;		// bにfalseをセット.
	}

	// 整数値, 文字列, 実数, 真偽値をフィールドにセット.
	public void set(int i, String str, float f, boolean b) {	// メソッドset(int i, String str, float f, boolean b)の定義.
		super.set(i, str);	// super.set(BaseClassのset)で, iとstrをセット.
		this.f = f;		// this.fにfをセット.
		this.b = b;		// this.bにbをセット.
	}

	// 各フィールドを出力.
	public void output() {	// メソッドoutputの定義
		super.output();	// super.output(BaseClassのoutput)で, iとstrを出力.
		System.out.println("f = " + f + ", " + "b = " + b);	// fとbを出力.
	}

}

File: cs/newModifier/newModifier/src/newModifier/newModifier/DerivedClass.cs
// 整数値と文字列を持つ派生クラス
class DerivedClass : BaseClass  // BaseClassの派生クラスDeriveClassの定義.
{

    // メソッドの定義
    // 初期化処理
    public DerivedClass() : base()   // コンストラクタDerivedClass()の定義.(親クラスのコンストラクタを呼ぶ.)
    {

    }

    // 初期化処理(指定の値をセット.)
    public DerivedClass(int i, string str) : base(i, str)   // コンストラクタDerivedClass(int i, string str)の定義.(親クラスのコンストラクタ(引数付き)を呼ぶ.)
    {

    }

    // 各フィールドを出力.
    new public void Output()    // メソッドOutputの定義(newで親クラスのメソッドを明示的に隠蔽する.)
    {
        // フィールドの値を出力.
        System.Console.WriteLine("(i, str) = (" + i + ", " + str + ")");    //iとstrの値を出力.
    }

}